doc/configuration.rst

```diff
@@ -483,6 +483,39 @@ Arguments:
 Used by:
   - `HIDRelayDriver`_
 
+HttpDigitalOutput
++++++++++++++++++
+A ``HttpDigitalOutput`` resource describes a generic digital output that can be
+controlled via HTTP.
+
+.. code-block:: yaml
+
+   HttpDigitalOutput:
+     url: http://host.example/some/endpoint
+     body_asserted: "On"
+     body_deasserted: "Off"
+
+The example assumes a simple scenario where the same URL is used for PUT
+requests that set the output state and GET requests to get the current state.
+It also assumes that the returned state matches either "On" or "Off" exactly.
+
+The `HttpDigitalOutputDriver`_ also supports more advanced usecases where the
+current state is fetched from another URL and is interpreted using regular
+expressions.
+
+Arguments:
+  - url (str): URL to use for setting a new state
+  - body_asserted (str): Request body to send to assert the output
+  - body_deasserted (str): Request body to send to de-assert the output
+  - method (str): HTTP method to use instead of PUT (the default) to set a new state
```

Comment:
```suggestion
  - method (str, default="PUT"): HTTP method to use instead of PUT to set a new state
```
Keeps things consistent with other resources and drivers.